### PR TITLE
Update to include result field and coin input

### PR DESCRIPTION
**Pull request by thodges-gh**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -95,6 +95,7 @@ export const calculateRate = async (baseAsset: string, interval: string) => {
     quoteAssets,
     price: price.toString(),
     volume: volume.toString(),
+    result: +(price.toString()),
```
Comment (thodges-gh) [inline src/index.ts]: I don't particularly like this myself. Does big.js have a cleaner way to produce just the number value? I couldn't find one in their docs.

Comment (Legogris) [reply src/index.ts]: How about just using the same as `result`? What's the use us casting it to a number as the receiving side can handle string-to-number conversion just fine? (Besides, `parseFloat` is more appropriate than using implicit conversions through operators IMO)

File: README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ zip -r cl-kaiko.zip .
 - Click Upload and select the `cl-kaiko.zip` file
 - Handler should remain index.handler
 - Add the environment variable:
-  - Key: API_KEY
+  - Key: CUBIT_API_KEY
```
Comment (Legogris) [inline README.md]: Is this conscious? I don't have a prefrence here, just making sure.

File: dist/package.json [hunk 3]
```diff
@@ -12,11 +12,11 @@
     "watch": "npx tsc -w"
   },
   "dependencies": {
-    "@google-cloud/logging-winston": "^0.11.1",
+    "@google-cloud/logging-winston": "^1.1.1",
     "big.js": "^5.2.2",
-    "moment": "^2.22.2",
+    "moment": "^2.4.0",
```
Comment (Legogris) [inline dist/package.json]: Why the downgrade?

**Comment by Legogris** [comment]
Just do I understand the motivation here: why `coin`? Moving forward, I think it's really a misnomer as for many assets to be referred to as "coins". Regardless, if this is due to keep API-consistency with other adapters, I'd rather prefer to remove the (IMO more semantically correct) `baseAsset` parameter and replace it with `coin` to just have one name for each parameter.

File: README.md (hunk 2 above)
Comment (thodges-gh) [reply README.md]: This is simply what you named the environment variable. I'm updating the docs to match. My preference would actually to be `API_KEY`. If you're okay with it, I can change it in the adapter.

File: dist/package.json (hunk 3 above)
Comment (thodges-gh) [reply dist/package.json]: :man_shrugging: It's automatic just from following the readme and running `npm install`.

**Comment by Legogris** [comment]
@thodges-gh I see now where the confusion comes from - I didn't notice first which branch was the base here! If you look at `master` it should be up to date wrt env variable, README and dependencies already.

**Comment by Legogris** [comment]
@thodges-gh Changed the base and rebased on `master`, hope the end-result is in line with the intentions.

**Pull request merged**